qt/python/mantidqt/_common.sip

```diff
@@ -984,6 +984,7 @@ public:
     void setFileExtensions(const QStringList &extensions);
     void findFiles(bool isModified);
     void isOptional(bool);
+	void setLastDirectory(const QString &lastDir);
```

Comment:
Just a small nitpick that the indentation of this line doesn't match the rest of the file